src/app/overview/home/home.component.ts

```diff
@@ -175,6 +175,23 @@ export class HomeComponent implements OnDestroy, OnInit {
     this.router.navigate(destination);
   }
 
+  goToAssessments(eventOrTodoItem) {
+    switch (eventOrTodoItem.type) {
+      case "feedback_available":
+        return this.goToAssessment(eventOrTodoItem.meta.activity_id, eventOrTodoItem.meta.context_id, eventOrTodoItem.meta.assessment_id);
+      case "review_submission":
+        return this.goToReview(eventOrTodoItem.meta.context_id, eventOrTodoItem.meta.assessment_id, eventOrTodoItem.meta.assessment_submission_id)
+      case "chat":
+        return this.goToChat(eventOrTodoItem);
+      case "assessment_submission_reminder":
+        return this.goToAssessment(eventOrTodoItem.meta.activity_id, eventOrTodoItem.meta.context_id, eventOrTodoItem.meta.assessment_id);
+
+      default: // event doesnt has type
```

Comment:
from the code below, it seems that the event also has type "event"

Reply:
oh, that one no need, cuz that's just required by `app-todo-card` component to identify which icon to display.
https://github.com/intersective/practera-app-v2/pull/672/files#diff-e84cb9b77c442630b4dbfa4135e113e2R67-R75